www/js/recommendations.js

```diff
@@ -47,11 +50,9 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
     $scope.clickSuggestion = function(id) {
       CommHelper.getSuggestion().then(function(result) {
```

Comment:
Why do we have two calls to `CommHelper.getSuggestion()`? Please unify.
This was a comment for the previous PR as well.